dashboard/Dockerfile

```diff
@@ -5,18 +5,11 @@
 FROM node:16-alpine
 COPY ./dashboard ./dashboard
 WORKDIR ./dashboard
-RUN npm install --production --legacy-peer-deps
```

Comment:
Try building this dockerfile and make sure it still works

Reply:
yup i think the dockerfile runs in cicd